src/schoology-extractor/data_extractor/main.py

```diff
@@ -0,0 +1,50 @@
+import os
+from lib import export_data
+from lib.users import Users
+from lib.assignments import Assignments
+from lib.sections import Sections
+from lib.submissions import Submissions
+from dotenv import load_dotenv
```

Comment:
I _think_ this is more like the usual standard: standard libraries, 3rd party, and then custom imports.

```suggestion
import os

from dotenv import load_dotenv

from lib import export_data
from lib.users import Users
from lib.assignments import Assignments
from lib.sections import Sections
from lib.submissions import Submissions
```

Reply:
I will change it